data.sql

```diff
@@ -6,3 +6,71 @@ VALUES('Agumon', '2020-02-03', 0, true,10.23),
      ('Gabumon', '2018-11-15', 2, true,8),
      ('Pikachu', '2021-01-07', 1, false,15.04),
      ('Devimon', '2017-05-12', 5, true,11);
+
+
+-- Query and Update animals table
+
+
+INSERT INTO animals(name,date_of_birth, escape_attempts,neutered,weight_kg,species) 
+VALUES('Charmander', '2020-02-08', 0, false,-11,NULL),
+     ('Plantmon', '2021-11-15', 2, true,-5.7,NULL),
+     ('Squirtle', '1993-04-02', 3, false,-12.13,NULL),
+     ('Angemon', '2005-06-12', 1, true,-45,NULL),
+     ('Boarmon', '2005-06-07', 7, true,20.4,NULL),
+     ('Blossom', '1998-10-13', 3, true,17,NULL),
+     ('Ditto', '2022-05-14', 4, true,22,NULL);
+
+
```

Comment:
- [ ] Kindly update the data as it's requested on the requirements 

![image](https://github.com/Bleu-Yves-Sopoude/vet-clinic-db/assets/93630700/0a6a3103-57c0-46d4-8b1c-ce05c7568ad3)